webserver.md

```diff
@@ -0,0 +1,280 @@
+{::options parse_block_html="true" /}
+
+<div class="row">
+
+<div class="large-12 columns">
+# Build Your Own Internet
+
```

Comment:
Thinking more on note 2 above, what if doing this on an AP was just a bonus?
I know you wrote this whole tutorial for the sake of the AP, but I think it might be cleaner. Thoughts?